docker/base/build.sh

```diff
@@ -313,6 +313,34 @@ for TARGET in $TARGETS; do
       CC=mips-linux-gnu-gcc-6 CXX=mips-linux-gnu-g++-6 GOOS=linux GOARCH=mips CGO_ENABLED=1 go build $V $X $TP $MOD "${T[@]}" --ldflags="$V $LD" $BM -o "/build/$NAME-linux-mips`extension linux`" $PACK_RELPATH
     fi
   fi
+  if ([ $XGOOS == "." ] || [ $XGOOS == "linux" ]) && ([ $XGOARCH == "." ] || [ $XGOARCH == "s390x" ]); then
+    if [ "$GO_VERSION" -lt 180 ]; then
```

Comment:
This check may not even be needed, since we dropped support for Go < 1.10. It doesn't hurt though.

Reply:
I'm not sure which version of go added support for s390x and powerpc64le cross compile but apparently it already worked in 1.11 (sorry fairly new to this)

Reply:
Looks like it has support for it since 1.7: https://golang.org/doc/go1.7

Reply:
Same for ppc64le

Reply:
Do you need a new PR or can it be fixed a after (-lt 170). ? I just want to know if the change is sane :P

Reply:
No need to create a new pr, could you just change that to `-lt 170`? (With a new commit)

Reply:
Just did - does that look right ?